app/views/users/registrations/signup_save_confirmation.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-large"><%=t '.heading' %></h1>
+
+    <p class="lede"><%=t '.lead_text_html', user_email: @user_email %></p>
+
+    <p><%=t('.expire_warning', expire_in_days: Rails.configuration.x.cases.expire_in_days) %></p>
```

Comment:
Another TODO comment here, about counting down days (or specifying the absolute date on which we'll discard your case)